apps/app/src/app/(app)/contributor-quest/page.tsx

```diff
@@ -0,0 +1,12 @@
+'use client';
+
+import { ContributorQuest } from '@/components/typeforms/contributor-quest';
+
+export default function Quest() {
+    return (
+      <div>
```

Comment:
Can you add this as well? Hides some unnecessary scrollbars

```suggestion
      <div className="overflow-hidden">
```